owtf/webapp/src/containers/WorklistPage/api.js

```diff
@@ -1,20 +1,22 @@
 import Request from "../../utils/request";
 import { API_BASE_URL } from "../../utils/constants";
 
+const options = {
+  headers: {
+    "Content-Type": "application/x-www-form-urlencoded;charset=UTF-8",
+    Authorization: "Bearer " + localStorage.getItem("token")
```

Comment:
doesn't this have to be in quotes? `"Authorization"`

Reply:
This is correct and is working fine when tested. The one with quotes is not working and also when I use auto-formatting it is removing the quotes.

Reply:
Hi @viyatb,
I searched this on google and found this link: https://stackoverflow.com/questions/60790742/vs-code-node-js-prettier-auto-formatting-messing-up-the-dictionary-key

Reply:
weird 👍